core-feature-pack/src/main/resources/content/bin/standalone.conf

```diff
@@ -50,23 +50,24 @@ fi
 # Specify options to pass to the Java VM.
 #
 if [ "x$JAVA_OPTS" = "x" ]; then
-   JAVA_OPTS="-Xms64m -Xmx512m -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=256m -Djava.net.preferIPv4Stack=true"
-   JAVA_OPTS="$JAVA_OPTS -Djboss.modules.system.pkgs=$JBOSS_MODULES_SYSTEM_PKGS -Djava.awt.headless=true"
-else
-   echo "JAVA_OPTS already set in environment; overriding default settings with values: $JAVA_OPTS"
-fi
+    JAVA_OPTS="-Xms64m -Xmx512m -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=256m -Djava.net.preferIPv4Stack=true"
+    JAVA_OPTS="$JAVA_OPTS -Djboss.modules.system.pkgs=$JBOSS_MODULES_SYSTEM_PKGS -Djava.awt.headless=true"
+
+    # Sample JPDA settings for remote socket debugging
+    #JAVA_OPTS="$JAVA_OPTS -agentlib:jdwp=transport=dt_socket,address=8787,server=y,suspend=n"
```

Comment:
that's a change in the script behaviour.
Now the debugging parameters are added only when JAVA_OPTS is not set in the environment.

Reply:
@rsearls there is still this part that needs to be updates, the commented settings for debugging should be after the if statement, not inside it.

Reply:
@jmesnil +1, this is regression of behaviour, I recommend to remove all changes of standalone.conf file.

Reply:
All standalone.* file changes withdrawn.   As marekkopecky recommends create an new jira for these files.